Framework/Core/src/runDataProcessing.cxx

```diff
@@ -1160,11 +1160,13 @@ int runStateMachine(DataProcessorSpecs const& workflow,
       auto bindResult = uv_tcp_bind(&serverHandle, (const struct sockaddr*)serverAddr, 0);
       if (bindResult != 0) {
         driverInfo.port++;
+        usleep(1000);
```

Comment:
I have already played on my system which shows these port finding issues with sleeps in some places, and they did not have any effect. The ratio I was seeing errors were the same. If you don't have a system where you can reproduce the issue and the sleep actually helps, I doubt that this is needed.